easybuild/easyconfigs/p/Python/Python-2.7.12-intel-2016b.eb

```diff
@@ -18,7 +18,8 @@ dependencies = [
     ('libreadline', '6.3'),
     ('ncurses', '6.0'),
     ('SQLite', '3.13.0'),
-    ('Tk', '8.6.5'),
+    # We don't build Tkinter as it requires an X11 stack
+    # ('Tk', '8.6.5'),
```

Comment:
@wpoely86 this should be done in a separate PR for sure, together with providing a separate easyconfig for `Tkinter` itself

also, we shouldn't make this change before EasyBuild v3.0.0